_episodes/first_example_recipe.md

```diff
@@ -0,0 +1,332 @@
+For additional reeds, please have a look at the recipe format description in the [ESMValTool manual](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/overview.html#recipe-section-diagnostics).
+
+## How to run ESMValTool
+
+Once you’ve set up your conda environment and installed ESMValTool (see episode #2 LINK) and set up your config-user.yml file to correctly match you local environment, (see episode #3 LINK), ESMValTool is invoked using a simple command:
+
+~~~source
+esmvaltool -c configuration recipe
+~~~
+
+To try your hand with a basic recipe, please work through this episode.
+
+## Introduction to the example recipe
+The recipe presented here is a simple, basic recipe that takes a single dataset and produces a time series plot.
+
+Please download the following recipe into your ESMValTool working area with the name: recipe_example.yml LINK
+
+> ## recipe_example.yml
+> ~~~YAML
+>  1    # ESMValTool
+>  2    # recipe_example.yml
+>  3    ---
+>  4    documentation:
+>  5      description: Demonstrate basic ESMValTool example
+>  6
+>  7      authors:
+>  8        - demora_lee
+>  9        - mueller_benjamin
+> 10        - swaminathan_ranjini
+> 11
+> 12      maintainer:
+> 13        - demora_lee
+> 14
+> 15      references:
+> 16        - demora2018gmd
+> 17        # Some plots also appear in ESMValTool paper 2.
+> 18
+> 19      projects:
+> 20        - ukesm
+> 21
+> 22    datasets:
+> 23      - {dataset: HadGEM2-ES, project: CMIP5, exp: historical, mip: Omon, ensemble: r1i1p1, start_year: 1859, end_year: 2005}
+> 24
+> 25    preprocessors:
+> 26      prep_timeseries: # For 0D fields
+> 27        annual_statistics:
+> 28          operator: mean
+> 29
+> 30    diagnostics:
+> 31      # --------------------------------------------------      
+> 32      # Time series diagnostics
+> 33      # --------------------------------------------------
+> 34      diag_timeseries_temperature:
+> 35        description: simple_time_series
+> 36        variables:
+> 37          timeseries_variable:
+> 38            short_name: thetaoga
+> 39            preprocessor: prep_timeseries
+> 40        scripts:
+> 41          timeseries_diag:
+> 42            script: ocean/diagnostic_timeseries.py
+> ~~~
+{: .solution}
+
+> ## Explore the recipe
+> Use the command and investigate the sample recipe.
+> ~~~bash
+> vim  recipe_example.yml
+> ~~~
+{: .challenge}
+
+Please note the following sections:
+  - documentation: lines 4-20
+
+    The documentation consists of the following information:
+    - description: a short description of the recipe
+    - authors: a list of authors (linked to esmvaltool/config-references.yml)
+    - maintainer: a list of maintainers (linked to esmvaltool/config-references.yml)
+    - references: a list of references (linked to a bibtexfile in esmvaltool/references with the same name)
+    - projects: a list of projects (linked to esmvaltool/config-references.yml)
+
+
+  - datasets: lines 22-23
```

Comment:
Same here as above: the information here seems slightly redundant since it has been mentioned in some detail already above. So maybe shorten the description above, or remove it there...